lr_face/data.py

```diff
@@ -178,14 +178,14 @@ def images(self) -> List[FaceImage]:
     @cache
     def pairs(self) -> List[FacePair]:
         """
-        Returns a list of `FacePair` instances from the images stored in this
-        dataset. Subclasses can override this method if the dataset has a
+        Subclasses can override this method if the dataset has a
         specific set of pairs associated with it (where not all images are used
-        for example).
+        for example). Returns an empty list otherwise, make_pairs should be
+        used on self.images
 
         :return: List[FacePair]
         """
-        return make_pairs(self.images)
```

Comment:
Ik begrijp niet waarom je dit hebt veranderd. Houdt dit in dat je voor elke dataset nu zelf de pairs()-methode moet implementeren? Dan ipv lege lijst een NotImplementedError teruggeven.

Reply:
heb de naam aangepast - deze wordt alleen gebruikt voor datasets waarbij een specifieke pairing bestaat. anders moet je idd zelf make_pairs aanroepen

Reply:
Yes, duidelijker nu.